x-pack/plugins/observability_solution/observability/public/pages/overview/components/header_menu/header_menu.tsx

```diff
@@ -10,6 +10,7 @@ import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { usePluginContext } from '../../../../hooks/use_plugin_context';
 import { useKibana } from '../../../../utils/kibana_react';
+// FIXME: import { HeaderMenuPortal } from '@kbn/observability-shared-plugin/public'
```

Comment:
I was not able to achieve this FIXME in this PR, without hard-to-debug test failures occurring.